Sources/ArcGISToolkit/Extensions/ArcGIS/AttachmentHelpers/AttachmentsFormElement.swift

```diff
@@ -27,8 +27,7 @@ extension AttachmentsFormElement : AttachmentsFeatureElement {
     /// This property will be empty if the element has not yet been evaluated.
     public var featureAttachments: [FeatureAttachment] {
         get async throws {
-            try await fetchAttachments()
-            return attachments.map { $0 as FeatureAttachment }
+            try await attachments.map { $0 as FeatureAttachment }
```

Comment:
This map is confusing. The cast in the transform closure is unconditional, so `attachments` itself could just be cast (`try await attachments as [FeatureAttachment]`). But since the cast is unconditional, the compiler knows that the cast will succeed, which makes me wonder if a cast is even needed.

Reply:
Good catch, thanks!